### ISSUE TITLE
After clicking the color picker the scroll doesn't work 

### ISSUE THREAD

**Issue by andrea78** [lbovet/wb0#5]
@lbovet After clicking the color picker, the scrolling function doesn't work in all known browsers but FF. I've tested on PC and notebook. So you can't zoom in or zoom out the canvas and you can't move the menu icons up and down. This is probably due to a script conflict.

How to reproduce it? 
1. Choose any color.
2. Click the color picker.
3. Now the scroll doesn't work.

**Comment by lbovet** [comment]
Unfortunately, I cannot reproduce the problem. Tried chrome and FF on linux and chrome and Edge on windows.
Also, can you move the palette feature proposal in another issue? They should be treated separately.

**Comment by andrea78** [comment]
I don't know if we are thinking the same.
By the color picker, I mean the dropper, not the color chooser or the color selector. 
Let's have a look below
![Screenshot 2021-09-24 at 19-01-37 chrome-dev-tools-color-picker webp (obraz WEBP, 884×520 pikseli)](https://user-images.githubusercontent.com/461774/134715328-aed543b3-0fd5-4b3a-86a7-e4e0e123a158.png)

**Comment by lbovet** [comment]
When the "dropper" is active, interaction is not possible with the page. That's a behavior of the browser that we cannot change.
Is this a problem?

**Comment by andrea78** [comment]
Unfortunately, that's not the point. If you click on the dropper, then the scroll does not work even if the dropper is turned off. You can't zoom in or zoom out the canvas and you can't move the menu icons up and down.

**Comment by lbovet** [comment]
I still cannot reproduce. What are the specs of your env?

**Comment by andrea78** [comment]
Sorry for my late reply. I've tested different specifications of my env and it's always the same. I don't think that's the cause.

**Comment by lbovet** [comment]
By specs, I meant, which browser on which OS ?

**Comment by andrea78** [comment]
Yeah, I see. The most popular chromium browsers for example Brave, Edge, etc. and OS Win 7, Win 10, macOS 10.15 Catalina.

**Comment by lbovet** [comment]
I tried to reproduce on Chrome and Edge under Windows 10 but the problem did not occur.

**Comment by MichMW** [comment]
Got the exact same problem. Scroll stops working after i used the color picker (the dropper)
